http/src/test/groovy/io/micronaut/http/HttpRequestSpec.groovy

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2017-2019 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.http
+
+import spock.lang.Specification
+
+class HttpRequestSpec extends Specification {
+
+    def "Converting to url and getting host and authority work as expected"() {
+        expect:
+        verifyAll {
```

Comment:
What is the purpose of `verifyAll` with asserts? Couldn't it just be `expect:` with each `HttpRequest... ==` on its own line?